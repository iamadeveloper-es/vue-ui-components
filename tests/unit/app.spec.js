import { createLocalVue, shallowMount } from "@vue/test-utils";
import App from '@/app';

describe('Component App', () => {
    const localVue = createLocalVue();

    const component = shallowMount(App, {
        localVue
    });

    beforeEach(() => {
        jest.clearAllMocks();
    });

    describe('... Definition', () => {

        test('The name of the component is app', () => {
            expect(App.name).toEqual('app');
        });

        test('Match snapshot', () => {
            expect(component.element).toMatchSnapshot();
        });
    });
});