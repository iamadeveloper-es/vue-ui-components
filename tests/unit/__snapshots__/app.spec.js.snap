// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component App ... Definition Match snapshot 1`] = `
<div
  id="app"
>
  <ui-view-dashboard-stub />
</div>
`;
